src/Core/Listener/CheckCustomLessFormat.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * (c) Toby Zerner <toby.zerner@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Flarum\Core\Listener;
+
+use Flarum\Core\Exception\ValidationException;
+use Flarum\Event\PrepareSerializedSetting;
+use Illuminate\Contracts\Events\Dispatcher;
+use Less_Exception_Parser;
+use Less_Parser;
+
+class CheckCustomLessFormat
+{
+    public function subscribe(Dispatcher $events)
+    {
+        $events->listen(PrepareSerializedSetting::class, [$this, 'checkCustomLessFormatBeforeSaving']);
+    }
+
+    public function checkCustomLessFormatBeforeSaving(PrepareSerializedSetting $event)
```

Comment:
I think just calling this method `check` or `safeguard` or something would suffice. No need to use such a long method name which repeats most of the class name.

Reply:
Sure. I moved the lot from the Provider and didn't rename it :innocent: